learn/guides/publish-packages-to-ballerina-central.md

```diff
@@ -112,6 +112,23 @@ Now, that you are ready to publish, execute the command below to publish the pac
 bal push
 ```
 
+### Publish successive package versions 
+If you require adding new features/improvements/fixes to a library package that you have already published to Ballerina central, 
+you are allowed to publish them under a new version, based on the [Semantic Versioning Specification](https://semver.org/).
+
+However, it's library developer's responsibility to be cautious when deciding new package versions
```

Comment:
```suggestion
>**Info:** It is the responsibility of the library developer to be cautious when deciding on the new package versions
```

Reply:
IMO we don't need to emphasise this paragraph as otherwise we'll have to do the same for the next paragraph as well. WDYT?

Reply:
+1 Let's keep it as normal text.